testutils/g109_samples.go

```diff
@@ -38,7 +38,7 @@ func main() {
 		fmt.Println(bigValue)
 	}
 }
-`}, 2, gosec.NewConfig()},
+`}, 1, gosec.NewConfig()},
```

Comment:
ditto

Reply:
same reason as above.
1 issue of G109 and other of G115.